discoverx/table_info.py

```diff
@@ -12,12 +12,34 @@ class ColumnInfo:
     classes: list[str]
 
 
+@dataclass
+class TagInfo:
+    tag_name: str
+    tag_value: str
+
+
+@dataclass
+class ColumnTagInfo:
+    column_name: str
+    tag_name: str
```

Comment:
Could use the TagInfo class here for each column ...

Reply:
It could, but that would require a different structure for columns.
My rationale was that having a flat list for each type of tag makes it easier to filter by tag, without having to look through the columns.